SparkPost/2web2ui
# FE-1238 - Hitting the escape key with the PlanPicker in the open state renders the error boundary

[FE-1238](https://sparkpost.atlassian.net/browse/FE-1238)

### What Changed
- Updates `downshift` to latest version

### How To Test
- To go `/onboarding/plan` and click on the custom plan select to open it, then hit the escape key to close it - the app shouldn't crash!
- Test other instances of components using `downshift` to ensure they still work as expected

### To Do
